src/FlaUI.WebDriver/Controllers/ActionsController.cs

```diff
@@ -303,6 +301,50 @@ private static async Task DispatchPointerAction(Session session, Action action)
             }
         }
 
+        private static Point GetCoordinates(Session session, Action action)
```

Comment:
Suggestion: Add a link to https://www.w3.org/TR/webdriver2/#dfn-get-coordinates-relative-to-an-origin for future reference

Reply:
As this is already referenced in the commit message, I think that is sufficient too.

Reply:
I think adding a link to the spec in source is a good idea. I'll do that in future!